po/it.po

```diff
@@ -116,80 +117,75 @@ msgstr "questo aiuto"
 
 #: newsboat.cpp:103
 msgid "remove unreferenced items from cache"
-msgstr ""
+msgstr "rimuovi elementi non referenziati dalla cache"
 
 #: newsboat.cpp:128
-#, fuzzy
 msgid "Files:"
-msgstr "File: "
+msgstr "Files: "
```

Comment:
Please remove the space, it looks like it's not needed anymore.

Reply:
Removing it